src/index.ts

```diff
@@ -1,11 +1,22 @@
 import './styles/style.scss';
+import { htmlElementConnector } from './utils';
 
 console.log('hello, world');
 
 const testMessage: string = 'TypeScript works';
 
 console.log(testMessage);
 
+const addEventListenerButton= document.getElementById("addEventListenerButton") as HTMLButtonElement;
+const connectorButton= document.getElementById("connectorButton") as HTMLButtonElement;
+const gradientColor = document.getElementById("changeClass") as HTMLButtonElement;
+
+function cursorOnButton(){
```

Comment:
take all the functions that respond to the UI events e.g. changeColor, getPersonalData  into a separate file. The New file may be called 'event-side-effects.ts'

Reply:
be sure to add export to the function you want to use in another file